libs/mock-block-dock/src/use-send-graph-value.ts

```diff
@@ -29,7 +29,7 @@ export const useSendGraphValue = ({
       (!sentInitially.current ||
         JSON.stringify(value) !== JSON.stringify(sentValue.current))
     ) {
-      graphService[valueName](value as any); // @todo how to maintain the union GraphValue is initially set to?
+      graphService[valueName]({ data: value as any }); // @todo how to maintain the union GraphValue is initially set to?
```

Comment:
This is an existing bug I had to fix to test properly – it was causing the actual content of messages to be `undefined` (e.g. the contents of `graph` in `initResponse`). Seemed like a good place to get it in.